src/openapi/transport/retry.js

```diff
@@ -150,12 +171,31 @@ TransportRetry.prototype.retryFailedCalls = function() {
 };
 
 /**
- * Disposes the underlying transport, the failed calls queue and clears retry timer.
+ * @protected
+ * @param transportCall
+ */
+TransportRetry.prototype.retryIndividualFailedCall = function(transportCall) {
+    transportCall.retryTimer = null;
+    const individualFailedCallsIndex = this.individualFailedCalls.indexOf(transportCall);
+    if (individualFailedCallsIndex >= 0) {
+        this.individualFailedCalls.splice(individualFailedCallsIndex, 1);
+    }
+    this.sendTransportCall(transportCall);
+};
+
+/**
+ * Disposes the underlying transport, the failed calls queue and clears retry timers.
  */
 TransportRetry.prototype.dispose = function() {
+    this.individualFailedCalls.forEach((failedCall) => {
+        if (failedCall.retryTimer) {
+            failedCall.retryTimer.clearTimeout();
+            failedCall.retryTimer = null;
+        }
+    });
     this.failedCalls.length = 0;
     if (this.retryTimer) {
-        this.retryTimer.clear();
+        this.retryTimer.clearTimeout();

```

Comment:
Hmm I don't understand this part. Afaik, this.retryTimer holds result of set setTimeout, which normaly returns number not instance with methods like clear and clearTimeout. 

I think I'm missing something :) Do we assign something else the setTimeout result to this property ?